### PR TITLE
Align report dates

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: packages/backend/src/peripherals/database/BalanceRepository.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ export class BalanceRepository extends BaseRepository {
       .insert(rows)
       .onConflict(['block_number', 'holder_address', 'asset_id'])
       .merge()
+      .returning('block_number')
```
Comment (sin6pi7) [inline packages/backend/src/peripherals/database/BalanceRepository.ts:54]: Why is this necessary?

Comment (sz-piotr) [reply packages/backend/src/peripherals/database/BalanceRepository.ts]: Reverted this changed and fixed it properly in #484

**Review by michalsidzej — APPROVED** [review]
